editor/src/clj/editor/app_view.clj

```diff
@@ -791,6 +791,7 @@
         (fn start-engine-build! []
           (assert (ui/on-ui-thread?))
           (cancel-engine-build!)
+          (println "START ENGINE BUILD")
```

Comment:
No other reasonable data we want in the logs?

Reply:
That was a debug print. I've removed it now.